Formula/llnode.rb

```diff
@@ -11,14 +13,10 @@ class Llnode < Formula
     sha256 "e047f606e4923900a3285acc8a352f847d387e877d5436ddf5cdbf37d256cf27" => :el_capitan
   end
 
-  depends_on "python@2" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build if MacOS.version <= :snow_leopard
```

Comment:
This can just be `depends_on "python@2" => :build`. The `if snow leopard` thing has been removed everywhere

Reply:
Good catch, accidentally caught this in a rebase.